src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -1,16 +1,110 @@
 (ns status-im2.contexts.wallet.collectible.view
   (:require [react-native.core :as rn]
             [quo2.core :as quo]
-            [re-frame.core :as rf]))
+            [utils.i18n :as i18n]
+            [status-im2.common.scroll-page.view :as scroll-page]
+            [status-im2.contexts.wallet.collectible.style :as style]
+            [status-im2.contexts.wallet.common.temp :as temp]))
 
 (defn view
   []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
-
-   [quo/text {} "COLLECTIBLES PAGE"]
-   [quo/divider-label]
-   [quo/button {:on-press #(rf/dispatch [:navigate-back])}
-    "NAVIGATE BACK"]])
+  (let [{:keys [name description image collection-image traits]} temp/collectible-details]
+    [scroll-page/scroll-page
+     {:navigate-back? true
+      :height         148
+      :page-nav-props {:type        :title-description
+                       :title       name
+                       :description description
+                       :right-side  [{:icon-name :i/options
+                                      :on-press  #(js/alert "pressed")}]
+                       :picture     image}}
+     nil
```

Comment:
what is nil being set on here? it's required or we can make this function/component multi-arity instead? 🤔

Reply:
Multi-arity doesn't work well with re-frame form-2 components, because they called only once and result is cached. But I passed sticky header component as a prop instead of making it an argument and that works well.